main.py

```diff
@@ -1,15 +1,16 @@
+"""This module is a function based trading bot"""
+import time
+import datetime
+import multiprocessing
+import functools
+import configparser
 from tradingview_ta import *
```

Comment:
It is recommended to include only needed modules when using 'from... import...' statement\
Else, 'import... as...' statement can be used for importing entire package and providing ease of access